nibabel/streamlines/tractogram.py

```diff
@@ -651,29 +650,46 @@ def apply_affine(self, affine):
         ----------
         affine : 2D array (4,4)
             Transformation matrix that will be applied on each streamline.
+        lazy : True, optional
```

Comment:
Yes, but I've put some thought into what you proposed: merging `LazyTractogram` and `Tractogram`. That might actually just work better than I expected with the advantage of offering a clearer interface. I'll try to merge them tonight.